source/server/options_impl.cc

```diff
@@ -22,6 +22,22 @@
 
 namespace Envoy {
 namespace {
+std::string prefixLogFormatWithLocation(const std::string& log_format) {
```

Comment:
`absl::StrReplaceAll` does not honor `%%`, e.g. output for `"%%v %v"` log format would be different.
I'm fine to change it if it's acceptable risk.

Reply:
I think it works with this: `absl::StrReplaceAll(foo, {{"%v", "[%g:%#] %v"}, {"%%", "%%"}});` because `%%` is an earlier replacement. This passes, which I think is the desired behavior: 
```
  std::string foo = "%%v %v";
  auto result = absl::StrReplaceAll(foo, {{"%v", "[%g:%#] %v"}, {"%%", "%%"}});
  EXPECT_EQ(result, "%%v [%g:%#] %v");
```

Reply:
Oh, fancy. Will do.